packages/jest-cli/src/__tests__/__snapshots__/notify_reporter.test.js.snap

```diff
@@ -31,10 +31,6 @@ Array [
 
 exports[`test change 1`] = `
 Array [
-  Object {
-    "message": "3 tests passed",
-    "title": "100% Passed",
-  },
```

Comment:
Should there be some snapshots added?

Reply:
My understanding is that this is a result of the `firstRun: i === 0` logic which is causing new snapshots in the the `change` testMode scenarios, due to me adding a new aggregation result to the `notifyEvents` array, so I believe this is expected?